<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form_Note</class>
 <widget class="QWidget" name="Form_Note">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>360</width>
    <height>412</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>360</width>
    <height>412</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>360</width>
    <height>412</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>13</x>
     <y>7</y>
     <width>331</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Sans Serif'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:20pt; color:#f81000;&quot;&gt;CATATAN&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>65</y>
     <width>57</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>Operator</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>105</y>
     <width>57</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>Judul</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>161</y>
     <width>57</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>Tipe</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>192</x>
     <y>161</y>
     <width>57</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>Channel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>208</y>
     <width>57</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>Catatan</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="edit_opt">
   <property name="geometry">
    <rect>
     <x>72</x>
     <y>60</y>
     <width>271</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="edit_judul">
   <property name="geometry">
    <rect>
     <x>72</x>
     <y>100</y>
     <width>271</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="combo_tipe">
   <property name="geometry">
    <rect>
     <x>72</x>
     <y>156</y>
     <width>101</width>
     <height>25</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Waveform</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Spektrum</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="combo_ch">
   <property name="geometry">
    <rect>
     <x>261</x>
     <y>156</y>
     <width>82</width>
     <height>25</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Ch. 1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Ch. 2</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Ch. 3</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Ch. 4</string>
    </property>
   </item>
  </widget>
  <widget class="QPlainTextEdit" name="catatan">
   <property name="geometry">
    <rect>
     <x>72</x>
     <y>203</y>
     <width>271</width>
     <height>171</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_save">
   <property name="geometry">
    <rect>
     <x>263</x>
     <y>380</y>
     <width>80</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>SAVE</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_close">
   <property name="geometry">
    <rect>
     <x>176</x>
     <y>380</y>
     <width>80</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>CLOSE</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
